src/app/core/terms/terms-txt.ts

```diff
@@ -0,0 +1,3 @@
+export const termsObj = {
+  version: '0.1'
```

Comment:
Can we store the text in here?

Reply:
@kewde Is there any reason for that ? I believe we can make the text static in html and as we have versioning stuff so whenever we change in text we can just update the version ?

Thanks

Reply:
The text and its version number form one entity, so it would be nice to have them next to eachother so we don't forget to update the version number.